source/nanoFramework.Hardware.Esp32/nanoFramework.Hardware.Esp32.nfproj

```diff
@@ -44,6 +44,9 @@
     <NFMDP_CMD_LINE_OUTPUT>false</NFMDP_CMD_LINE_OUTPUT>
     <Name>nanoFramework.Hardware.Esp32</Name>
   </PropertyGroup>
+  <PropertyGroup Label="nanoFramework">
```

Comment:
This shouldn't be set. It's valid for Interop.

This is a class library therefore a VIP in nF. 😉 
The stubs for the native classes are already and always generated.

You need to grab the new ones (for this new classes) and also the modified declaration to port over/update what is already in the nf-interpreter repo @ targets\FreeRTOS_ESP32\ESP32_WROOM_32\nanoCLR\nanoFramework.Hardware.ESP32.